samples/dashboards/cypress/component-with-no-connector/demo.html

```diff
@@ -1 +1,3 @@
-<div id="container">
+<script src="https://code.highcharts.com/grid/grid-pro.js"></script>
+
+<div id="container">
```

Comment:
We should not mix loading modules in two ways. You should import grid-pro module from es-modules.

It will work only in utils, not after deploy.